lua/cmake-tools/terminal.lua

```diff
@@ -542,7 +542,7 @@ end
 local get_command_handling_on_exit = function()
   return "echo $? > "
     .. get_last_exit_code_file_path() -- write exitcode to file
-    .. " && rm "
+    .. " && \\rm -f "
```

Comment:
Why is there a `\\`?

Reply:
Oh, this makes sense.

Reply:
this breaks the fish shell.
I'd suggest to use the `command rm -f` alternative instead